src/common/utils/api.ts

```diff
@@ -39,6 +39,12 @@ const performRequest = async (query: string, parameters: IQueryObject = {}, opti
     ...options.headers,
     Authorization: options.user ? `Bearer ${options.user.access_token}` : '',
   };
+
+  // requests sometimes fail to resolve without error when Authorization is set to ''
+  if (!options.user) {
+    delete headers.Authorization;
+  }
+
```

Comment:
These statements sort of contradict each other, as we are always adding something (the Authorization header), and then removing it in some cases afterwards.

Maybe we should just do `if options.user`; add `Authorization` header?

Reply:
sure